modal/file_io.py

```diff
@@ -188,6 +190,12 @@ async def _wait(self, exec_id: str) -> bytes:
                 raise
         return output
 
+    def _parse_list_output(self, output: bytes) -> list[str]:
```

Comment:
On further thought, I don't think it makes sense for this to be pulled out into its own function. It's only used in `ls`, so I think you should keep it there.